### ISSUE TITLE
Generation speed quite different 

### ISSUE THREAD

**Issue by r530044129** [bytedance/MVDream#18]
From my observation, the gereration speed are quite different between different prompt.

Some quick just need 1 hour maybe(GPU 3090), but some need cost 20+ hours.

Is that right that some prompt will cost far more time than other? Or is most of the generation time the same, no matter what prompt it is?

**Comment by seasonSH** [comment]
Are you talking about the 3D generation with threestudio repo? 
The generation speeds would be affected by the prompt because InstantNGP does pruning when sampling along rays. However, I have never observed such a big difference between prompts.
For example, the config without shading takes me 40-60 min on V100, never goes up to 2hours.

Can you share the prompt and results that cost you 20 hours?

**Comment by r530044129** [comment]
yeah, it's 3d generation. You can try this one: 'Rhinoceros statue, art, Rhinoceros head', which cost me 12 hours and just arrive 8537 iter

Epoch 0: : 8539it [12:12:08,  5.14s/it]

Im not sure is that because my PC's problem or some GPU memory accumulation or something lead to such slow generation. I can feel very big difference between prompt. If there are some commom prompt, such as 'cat' \ 'dog' \ 'chair', it will be quick, but if there are some uncommon prompt, it will cost me a lot of time.
